pkg/device_plugin/device_plugin.go

```diff
@@ -53,8 +53,26 @@ func (m *SGXDevicePlugin) ListAndWatch(e *devicepluginapi.Empty, s deviceplugina
 // Allocate which return list of devices.
 // Allocate implements DevicePluginServer interface.
 func (m *SGXDevicePlugin) Allocate(ctx context.Context, reqs *devicepluginapi.AllocateRequest) (*devicepluginapi.AllocateResponse, error) {
+	var mounts []*devicepluginapi.Mount
 	var devices []*devicepluginapi.DeviceSpec
 
+	if sgx.EnableAesm {
+		for path, exist := range sgx.AllMountPoints() {
+			if path != sgx.AESMSocket {
+				continue
+			}
+			if exist {
```

Comment:
We have already check aesm socket inside init() on startup, so just remove this `if exist` .